components/org.wso2.carbon.identity.remotefetch.core/src/test/resources/testng.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the license header.

Reply:
[Fixed](https://github.com/wso2-extensions/identity-fetch-remote/pull/13/commits/21eee2a3e6dc27743cc668f4f3aaf303ece32859)